resources/translations/fr_FR/troubleshooting/bed_adhesion_problems.md

```diff
@@ -1,52 +1,52 @@
+* ...grossier ! Avec un plateau légèrement rugueux,la surface de contact sur le plateau et l'impression est considérablement augmentée, contrairement à un plateau parfaitement lisse. Pour cette raison, certaines imprimantes sont équipées de verre dépoli ou de métal anodisé.
 
 Le matériau de la plaque de montage a également un impact important sur les propriétés adhésives. Celles-ci peuvent varier considérablement en fonction des matériaux d'impression. Les matériaux les plus courants sont le verre, l'acrylique, l'aluminium, l'acier ou l'aluminium anodisé. Chacun d'eux adhère à des matériaux différents. Les plaques de montage en plastique peuvent adhérer chimiquement à votre matériau. D'autres matériaux se caractérisent par une surface rugueuse qui augmente la surface de contact.
 
-Il existe également des méthodes d'adhésion par lit supplémentaire que vous pouvez appliquer à n'importe quelle plaque de construction :
+Il existe également des méthodes d'adhérence par lit supplémentaire que vous pouvez appliquer à n'importe quelle plaque de construction :
 * Les bâtons de colle à papier génériques fonctionnent très bien. Bien que la colle sèche rapidement, elle produit une surface rugueuse avec une surface accrue pour que l'impression y adhère.
 * Le ruban de masquage a un effet similaire, puisqu'il a une surface rugueuse faite de papier, mais contrairement au papier, il reste parfaitement plat sur la plaque de montage. Les types de rubans de masquage plus épais, conçus pour une utilisation extérieure, sont les plus efficaces.
-* La laque pour cheveux fonctionne de la même manière que la colle, en ce sens qu'elle laisse une surface rugueuse à grain fin sur la plaque de construction. Il est très facile de l'appliquer uniformément.
-* Appliquez du papier de verre sur la plaque de montage pour la rendre plus rugueuse et augmenter la surface de contact. Ne la rendez pas trop rugueuse, sinon vous aurez une surface inégale pour imprimer. Pour ce faire, utilisez un papier de verre à grain très fin, d'un grain de 500 ou plus.
-* Des "feuilles d'adhérence" spécialisées sont vendues par de nombreux vendeurs d'imprimantes 3D, qui peuvent être tendues sur la plaque de construction. Elles fonctionnent de la même manière que le ruban adhésif, mais sont conçues pour durer plus longtemps. Certaines sont faites d'un plastique qui adhère chimiquement au matériau imprimé également.
+* La laque pour cheveux fonctionne de la même manière que la colle, en ce sens qu'elle laisse une surface rugueuse à grain fin sur le plateau. Il est très facile de l'appliquer uniformément.
+* Appliquez du papier de verre sur l plateau pour la rendre plus rugueuse et augmenter la surface de contact. Ne la rendez pas trop rugueuse, sinon vous aurez une surface inégale pour imprimer. Pour ce faire, utilisez un papier de verre à grain très fin, d'un grain de 500 ou plus.
```

Comment:
`le plateau?`

Reply:
Yes , "le plateau"  but should be "Utilisez du papier de verre sur le plateau pour le rendre plus rugueux"   Well I need to change all this section.